twitterscraper/query.py

```diff
@@ -223,3 +221,65 @@ def query_tweets_from_user(user, limit=None):
         len(tweets), user))
     return tweets
 
+
+def query_user_page(url, html_response=True, retry=10):
```

Comment:
the `html_response` argument is redundant here. In line 274 it is not specified and it seems it is also not specified in the rest of the added code where the query_user_page() method is used. 
The reason why it was necessary for the `query_single_page()` method is because the scraped tweets can be retrieved in the form of a html page or in the form of a JSON list, which requires different behaviour. The same can not be said for scraping of an user page for info.

Reply:
Done